p11nethsm.example.conf

```diff
@@ -38,9 +38,18 @@ slots:
       username: "operator"
       # If the password starts with `env:`, it will obtain the password from an environment variable:
       # password: "env:LOCALHSMPASS"
+      #
+      # If the field is not provided, it is expected to be given through `C_Login` function with the
+      # `CKU_USER` userType
       password: "localpass"
```

Comment:
Maybe by default username and password should be commented (inactive) because the default values will be wrong for most users.

Reply:
Is this really changed as intended? To me it looks mixed up: First password, then username than passwords are explained again.

![grafik](https://github.com/user-attachments/assets/33f4877b-2a9a-4cfa-b97c-eb9488800c59)